### PR TITLE
Create header component

### LINKED ISSUE #40: Create a header component
- We need a header component that can be used on the three pages
- it should follow the design on [figma](https://www.figma.com/file/4qFFHuORhrqvQA1bleIRa4/TCL-44?node-id=0%3A1)

### PR DESCRIPTION
**Pull request by redapy**
## Description
This PR creates a `Header` component that will be used in the three pages
- I Created a `Header` component and use it on our three pages.
- in the `Home` page it takes a `isHome` prop that will be used to style the header differently if we are in the home page.
- I changed the `Navbar` color to match the header and buttons' color.
- I moved the styles in `app.css` to `index.css`, I thought it's better to have only one stylsheet.

## Related Issue
closes #40 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before
before

### After
after

## Testing Steps / QA Criteria
- From your terminal, pull down this branch with `git pull origin rb-create-header-component` and check that branch out with git checkout `b-create-header-component`.
- compare the changes with the Figma file.

### CODE REVIEW

**Comment by JimeBlue** [comment]
Great job Reda! The header looks very well in the 3 pages and according to the figma design.

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const Header = ({ title, imageSrc, isHome }) => {
+  let headerStyles;
+  let titleStyles;
+  if (isHome) {
+    headerStyles = 'bg-yellow-400 w-full text-center relative h-32 mb-32';
+    titleStyles = ' font-bold text-2xl w-1/4 mx-auto mt-4';
```
Comment (hellodeborahuk) [inline src/components/Header.js:6]: I think we need to change the width on mobile view. It puts the title on too many lines.

title spacing

**Review by hellodeborahuk — COMMENTED** [review]
Great job Reda. I like how you've implemented the header and title styles in an if statement.

File: src/components/Header.js (hunk 1 above)
Comment (redapy) [reply src/components/Header.js:6]: @JimeBlue will tackle this problem in another issue

**Pull request merged**